lib/db/files_db.dart

```diff
@@ -227,6 +238,20 @@ class FilesDB {
     ];
   }
 
+  static List<String> addMagicMetadataColumns() {
+    return [
+      '''
+        ALTER TABLE $table ADD COLUMN $columnMMdEncodedJson Text DEFAULT '{}';
```

Comment:
`TEXT`

Reply:
Do we need to store the encoded JSON separately since it can be derived from the other columns anyway?

Reply:
This is required to make client future proof. If we start deriving this information from other columns, then older clients won't be able to support new metadata information.